mne/source_estimate.py

```diff
@@ -2591,6 +2681,172 @@ def compute_morph_matrix(subject_from, subject_to, vertices_from, vertices_to,
     return morpher
 
 
+@verbose
+def compute_morph_sdr(mri_from, mri_to,
+                      niter_affine=(100, 100, 10),
+                      niter_sdr=(5, 5, 3),
+                      voxel_size=None,
+                      verbose=None):
+    """Get a matrix that morphs data from one subject to another.
```

Comment:
I would say "Compute a non-linear volume morphing operator using SDR"

Is it linear or non-linear operator on the actual STC data? In other words, do we have that:
```
stc1.morph_precomptued(...) + stc2.morph_precomputed(...) == (stc1 + stc2).morph_precomputed(...)
```
Based on your description I think the answer is "no". But if we do, put "linear" in instead of "nonlinear".

Reply:
yes, it's non-linear - the text is a relic from cp